cryptofeed/exchanges/huobi.py

```diff
@@ -32,6 +32,7 @@ class Huobi(Feed):
         L2_BOOK: 'depth.step0',
         TRADES: 'trade.detail',
         CANDLES: 'kline',
+        TICKER = 'ticker'
```

Comment:
probably should be : `TICKER: 'ticker'`

Reply:
You are totally right, somehow it slipped me

Reply:
You're totally right, somehow I missed it